study-builder/studies/rarex/consent-lar-assent.conf

```diff
@@ -1180,35 +968,55 @@
             }
           },
           "blockType": "QUESTION",
-          "shownExpr": """user.studies["rarex"].forms["LAR_CONSENT_ASSENT"].questions["LAR_ASSENT_ABLE_READ"].answers.hasOption("ASSENT_ABLE_READ_NO")"""
+          "shownExpr": """user.studies["rarex"].forms["LAR_CONSENT_ASSENT"].questions["LAR_ASSENT_ABLE_READ"].isAnswered()"""
```

Comment:
We're now using only one set of questions. When they answer the question these blocks will show up.